src/data_standardization/standardize_point_data.py

```diff
@@ -9,19 +9,55 @@
     os.path.abspath(__file__))
 BASE_FP = os.path.dirname(CURR_FP)
 
+def process_row(row: dict, row_config: dict) -> dict:
+    new_row = OrderedDict()
+    if "category" in row_config and row_config['category']:
+        new_row['category'] = row[row_config['category']]
+    if "notes" in row_config and row_config['notes']:
+        new_row['notes'] = row[row_config['notes']]
+    if "feat_agg" in row_config and row_config['feat_agg']:
+        new_row['feat_agg'] = row_config['feat_agg']
+    if "value" in row_config and row_config['value']:
+        # value must be numeric
+        val = row[row_config['value']]
+        if type(val) not in (int, float):
+            val = pd.to_numeric(val).item()
+        new_row['value'] = val
+    return(new_row)
 
-def read_file_info(config, datadir):
+def read_file_info(config: data.config.Configuration,
+                   datadir: str) -> None:
+    """
+    Reads information from additional point based feature files, processes, validates and writes them
+    Args:
+        config: project configuration object
+        datadir: filepath for point based features
+    Returns: None, writes output directly
+
+    """
 
     points = []
     for source_config in list(config.data_source):
 
-        print("Processing {} data".format(source_config['name']))
+        print("Processing {} data".format(source_config['filename']))
         csv_file = source_config['filename']
         filepath = os.path.join(datadir, 'raw', 'supplemental', csv_file)
         if not os.path.exists(filepath):
-            raise SystemExit(csv_file + " not found, exiting")
+            # automatically check the raw/crashes directory
+            print(f"{filepath} not found, trying 'raw/crashes' directory")
+            filepath = os.path.join(datadir, 'raw', 'crashes', csv_file)
+            if not os.path.exists(filepath):
+                raise SystemExit(filepath + " not found, exiting")
 
         df = pd.read_csv(filepath, na_filter=False)
+
```

Comment:
Take out the commented out code unless you think you'll need it, otherwise add a comment about it